blog/2019-10-29-dhis2-and-docker.md

```diff
@@ -7,6 +7,9 @@ author_image_url: https://avatars.githubusercontent.com/u/12664326?s=400&v=4
 tags: [docker]
 ---
 
+[//]: # (TODO Should we mention an update date?)
```

Comment:
The blog post is from `October 29, 2019` which does not seem recent to me. Maybe it should not show up in https://developers.dhis2.org/blog recent posts.

Not sure if I would update a blog post. To me that is a one time "article". If there is information that we discover isn't true I would update it. But I would not keep a blog post up to date for 5 years or more 😅 

We should extract information that is useful and still applies to the Docker section in https://github.com/dhis2/dhis2-core.

Reply:
@teleivo I added an update note to the top of the blog post - https://github.com/dhis2/developer-portal/pull/105/commits/f5933569dd31327795fe8d8459478e7acfb7a291

And extracted some of the info to the dhis2-core readme - https://github.com/dhis2/dhis2-core/pull/12831

Do you think we should extract something more?

Reply:
Adding that comment to the blog post makes sense. I would also remove the blog post from recent posts 😄  Its not recent and outdated. @Topener whats your opinion on the above?

Reply:
The recent blogposts are indeed not recent, but it's the last 15 (or so) articles published there. One way to get rid of it, is writing more (I'm working on it).

I wouldn't update the old blogpost with new content, but a warning it's outdated is for sure a good one! Preferably with a link to new content in that case, either a guide or a new blogpost

Reply:
@Topener has done what you suggested. Is this change ready to be merged from your point of view?